### PR TITLE
Feature #606 - Show hint to interact with extension

### LINKED ISSUE #606: Show hint when user interaction in extension is required
When a `getIdentities` or `signAndPost` request is sent, it is not clear to users that they need to interact with the extension in order to continue.

Can we print a hint like "Please authorize in NEUMA" (the new name of the extension)? Ideally this should be written into the button that was triggering the event. But this is probably too long. Can we still give it a try? Other ideas?

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will add Billboard component and context to show full app message.

Closes #606.

[Video](https://drive.google.com/file/d/1jGckEvTvmWP7vyzppmPL984ZlAk-8_dd/view)

### CODE REVIEW

**Comment by albertandrejev** [comment]
Better message and UI layout is required.

File: packages/bierzo-wallet/src/components/BillboardMessage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Block, Typography } from "medulas-react-components";
+import React from "react";
+
+function BillboardMessage(): JSX.Element {
+  return (
+    <Block bgcolor="white" padding={2}>
+      <Typography variant="h6">Please interact with NEUMA extension before continue.</Typography>
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/components/BillboardMessage/index.tsx]: The spelling of NEUMA was just changed to "Neuma". I suggest here:

> Please authorize request in Neuma to continue.

File: packages/bierzo-wallet/src/routes/payment/index.tsx [hunk 2]
```diff
@@ -90,6 +93,8 @@ const Payment = (): JSX.Element => {
       console.error(error);
       toast.show("An error ocurred", ToastVariant.ERROR);
       return;
+    } finally {
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/routes/payment/index.tsx]: I think the `return;` must be removed in order to close this in the exception case

File: packages/bierzo-wallet/src/routes/registerName/index.tsx [hunk 3]
```diff
@@ -110,6 +121,8 @@ const RegisterUsername = (): JSX.Element => {
       console.error(error);
       toast.show("An error ocurred", ToastVariant.ERROR);
       return;
+    } finally {
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/routes/registerName/index.tsx]: Remove `return;` as above

File: packages/bierzo-wallet/src/utils/test/persona.ts [hunk 4]
```diff
@@ -16,8 +16,8 @@ export const submitExtensionSignupForm = async (page: Page, password: string): P
   await clickCreatePersona(page);
 
   // Fill the form
-  await page.type(`input[name="${PASSWORD_FIELD}`, password);
-  await page.type(`input[name="${PASSWORD_CONFIRM_FIELD}`, password);
+  await page.type(`input[name="${PASSWORD_FIELD}"]`, password);
+  await page.type(`input[name="${PASSWORD_CONFIRM_FIELD}"]`, password);
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/utils/test/persona.ts]: This is fixed on master already. You can rebase to remove the diff from here

**Pull request merged**